### PR TITLE
allow more fields in servingruntime container spec

### PR DESCRIPTION
**Pull request by Suresh-Nakkeran**
Signed-off-by: Suresh Nakkeran <suresh.n@ideas2it.com>

Issue #2095 

In servingruntime, a custom Container spec has been used. This PR is to change custom Container spec to k8s v1.Container spec and make use of all fields.

### CODE REVIEW

File: pkg/apis/serving/v1alpha1/servingruntime_types.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ type ServingRuntimePodSpec struct {
 	// Cannot be updated.
 	// +patchMergeKey=name
 	// +patchStrategy=merge
-	Containers []Container `json:"containers" patchStrategy:"merge" patchMergeKey:"name" validate:"required"`
```
Comment (njhill) [inline pkg/apis/serving/v1alpha1/servingruntime_types.go:73]: @Suresh-Nakkeran the `Container` struct above (line 41) should also now be removed.

**Comment by njhill** [comment]
@pvaneck some changes will be needed on the model-mesh controller side too for this of course.

File: pkg/apis/serving/v1alpha1/servingruntime_types.go (hunk 1 above)
Comment (Suresh-Nakkeran) [reply pkg/apis/serving/v1alpha1/servingruntime_types.go:73]: Removed. Thanks!

**Pull request merged**